javascript-carousel/javascript-carousel/main.js

```diff
@@ -0,0 +1,64 @@
+var $icons = document.querySelectorAll('.size2');
+var $image = document.querySelector('img');
+var imageUrl = ['images/001.png', 'images/004.png', 'images/007.png', 'images/025.png', 'images/039.png'];
+var x = 0; // Used to keep track which image to display
+$image.setAttribute('src', imageUrl[x]);
+
+var counter = 3; // After user selects change, goes back to narmal after 3 secs
```

Comment:
Try to rewrite your code to only keep track of the `index` of the active image. The dot element and image url index are the same number.